core/block_render_svg_vertical.js

```diff
@@ -1606,7 +1606,8 @@ Blockly.BlockSvg.prototype.renderDefineBlock_ = function(steps, inputRows,
         input.connection.targetBlock().getHeightWidth().width +
         Blockly.BlockSvg.DEFINE_BLOCK_PADDING_RIGHT;
   } else {
-    rightSide = Blockly.BlockSvg.MIN_BLOCK_X_WITH_STATEMENT;
+    rightSide = Math.max(Blockly.BlockSvg.MIN_BLOCK_X_WITH_STATEMENT, rightSide)
```

Comment:
Add a comment that this case handles the block being rendered as an insertion marker (in all other cases there should be a target block connected on the statement input).